packages/react-examples/src/react/Pickers/Pickers.doc.tsx

```diff
@@ -5,11 +5,13 @@ import { TagPickerCustomRemoveIconExample } from './TagPicker.CustomRemoveIcon.E
 import { IDocPageProps } from '@fluentui/react/lib/common/DocPage.types';
 import { TagPickerBasicExample } from './TagPicker.Basic.Example';
 import { TagPickerInlineExample } from './TagPicker.Inline.Example';
+import { TagPickerFocusZoneExample } from './TagPicker.FocusZone.Example';
 
 const TagPickerExampleCode = require('!raw-loader?esModule=false!@fluentui/react-examples/src/react/Pickers/TagPicker.Basic.Example.tsx') as string;
 const TagPickerInlineExampleCode = require('!raw-loader?esModule=false!@fluentui/react-examples/src/react/Pickers/TagPicker.Inline.Example.tsx') as string;
 const PickerCustomResultExampleCode = require('!raw-loader?esModule=false!@fluentui/react-examples/src/react/Pickers/Picker.CustomResult.Example.tsx') as string;
 const TagPickerCustomRemoveIconExampleCode = require('!raw-loader?esModule=false!@fluentui/react-examples/src/react/Pickers/TagPicker.CustomRemoveIcon.Example.tsx') as string;
+const TagPickerFocusZoneExampleCode = require('!raw-loader?esModule=false!@fluentui/react-examples/src/react/Pickers/TagPicker.FocusZone.Example.tsx') as string;
```

Comment:
I'm actually kinda against putting this on our docs site, or at least putting it there without strong warnings against using it.

While we've made the keyboard interaction work for this (albeit "work" in this case is worse than the no-focuszone UX), the screen reader experience is still all wrong. I don't want to mislead authors into thinking this is something they should do vs. something we still support for legacy reasons :P